CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@ This document describes changes between each past release.
 6.1.0 (unreleased)
 ------------------
 
+**Protocol**
+
+- The history entries for deletions now provide the deleted object instead of
+  just the tombstone (fixes #1135)
+
+Protocol is now at version **1.15**. See `API changelog`_.
```

Comment:
The version here doesn't match the version on the __init__ file (should be 1.16 I guess)